### PR TITLE
Disable eval button when waiting for response

### PR DESCRIPTION
**Pull request by brendanzab**

### CODE REVIEW

File: src/client/Main.elm [hunk 1]
```diff
@@ -111,13 +117,13 @@ update msg model =
     case msg of
         EvalRequested ->
             -- TODO: Disable the eval button
```
Comment (Marwes) [inline src/client/Main.elm]: This TODO can be removed now.

File: src/client/Main.elm [hunk 2]
```diff
@@ -163,16 +169,32 @@ editor model =
...
-                , button [ class "btn btn-primary pull-xs-right", onClick EvalRequested ]
-                    [ text "Eval" ]
+    let
+        evalResult =
+            case model.evalResult of
+                Pending ->
+                    "Waiting..."
+
+                Fail _ ->
+                    "Http error."
```
Comment (Marwes) [inline src/client/Main.elm]: The argument to `Fail` is never read from, I don't know if it is something that could be displayed to the user but if not the argument to `Fail` could be removed.

**Comment by brendanzab** [comment]
Fixed!

**Comment by Marwes** [comment]
Will try to deploy this tonight but I am going to see first if I can deployment and testing this a bit easier.

**Pull request merged**